spec/features/flights_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+describe "Flight", type: :feature do
+  before(:each) do
+    load "#{Rails.root}/spec/support/seed.rb"
+  end
+
+  before(:each) do
+    Seed.airports
+    Seed.flights
+  end
+  context "Search Flight" do
+    it "Search for flight without date " do
+      visit "/"
+      select "N'djili Airport Kinshasa, Congo", from: "departure_id"
+      select "Julius Nyerere International Airport Dar es Salam, Tanzania", from: "arrival_id"
+      fill_in('departure_date', with: "1990 - 11 - 1")
+      select 1, from: "seats"
+      click_button 'Search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.